DataAccess/ProductsData.xml

```diff
@@ -0,0 +1,33 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
```

Comment:
an easier way would be using json rather than xml, and you can do deserialize from it